clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11787,6 +11787,9 @@ def err_ivdep_declrefexpr_arg : Error<
 def warn_ivdep_redundant : Warning <"ignoring redundant Intel FPGA loop "
   "attribute 'ivdep': safelen %select{INF|%1}0 >= safelen %select{INF|%3}2">,
   InGroup<IgnoredAttributes>;
+def warn_ivdep_attribute_argument : Warning<
+  "'ivdep' attribute applies safelen value of %0 to the loop">,
```

Comment:
This warning message doesn't seem right?  It seems like we're just saying "this ivdep attribute is doing what it is supposed to".  Would the following be accurate?
``` 'ivdep' attribute safelen value of %0 has no effect ```
or something like that?  @premanandrao : Got ideas on wordsmithing?

Reply:
I like what you have.  Maybe: ``'ivdep' attribute safelen value of %0 has no effect on the loop`` ?

Reply:
Usually for wordsmithing, I just call on my bestest of wordsmithy buddies... @AaronBallman :D

Reply:
I'd go with `'ivdep' attribute with value %0 has no effect; attribute ignored` and skip the odd wording about "safelen" entirely (I don't know that people will know about the names of the attribute parameters, generally).

Reply:
Done